Gruntfile.js

```diff
@@ -225,6 +225,28 @@ var config = require('./core/server/config'),
             // ### grunt-shell
             // Command line tools where it's easier to run a command directly than configure a grunt plugin
             shell: {
+                submodules: {
+                    command: function () {
+                        grunt.log.writeln('SUBBY');
+
+                        var upstream = grunt.option('upstream') || process.env.GHOST_UPSTREAM || 'upstream';
+                        var update = 'git submodule update --init';
+
+                        grunt.log.writeln('Updating git submodules: ' + upstream);
+
+                        // `git submodule update` will clone repos as origin,
+                        // we need to change this to whatever the user uses as upstream (probably "upstream")
+                        if (upstream !== 'origin') {
+                            grunt.log.writeln('Changing remote to: ' + upstream);
+                            update += '; cd core/client; git remote rename origin upstream; ' +
```

Comment:
Should the `upstream` value be used in the rename? Eg.

update += '; cd core/client; git remote rename origin ' + upstream + '; ' +